docs/articles/faq/README.md

```diff
@@ -245,24 +245,36 @@ Finally, try updating TestCafe to the latest version to see if the problem persi
 When TestCafe does not receive a successful response from a server, it outputs the following error:
 
 ```text
-A request to https://www.example.com has failed.
-Use quarantine mode to perform additional attempts to execute this test.
+Failed to load the page at https://www.example.com.
+Increase the value of the "pageRequestTimeout" variable, enable the "retryTestPages" option, or use quarantine mode to perform additional attempts to execute this test.
 ```
 
-You can use [quarantine mode](../documentation/guides/basic-guides/run-tests.md#quarantine-mode) to complete the tests if this problem occurs infrequently.
+If your test produces unstable results, you can enable [quarantine mode](../documentation/guides/basic-guides/run-tests.md#quarantine-mode) to isolate it from the rest of the suite.
 
 However, we recommend that you determine the cause of this issue and address it.
 
 This error can occur in the following situations:
 
 #### The Web server is not responding
 
-Check if the Web and DNS servers are online and configured to accept requests to this URL.
+Confirm the successful resolution of the website's domain name. Check if the web application is online and configured to receive incoming requests.
+
+If TestCafe does not receive the webpage within *two minutes*, it throws an exception.
```

Comment:
I'd specify this fact, "if the TestCafe proxy..." yadda yadda 

Because it's hard for readers to distinguish b/w TestCafe the framework and TestCafe the test runner